zscaler_connector.py

```diff
@@ -118,7 +118,7 @@ def _process_html_response(self, response, action_result):
             split_lines = error_text.split('\n')
             split_lines = [x.strip() for x in split_lines if x.strip()]
             error_text = '\n'.join(split_lines)
-        except:
+        except Exception:
             error_text = "Cannot parse error details"
```

Comment:
Can we include the error message in the text or in debug_print?

Reply:
Yes, Added into debug_print.